x-pack/plugins/endpoint/server/services/resolver/common.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { ResolverPhase0Data, ResolverPhase1Data, ResolverData } from '../../../common/types';
+
+export const phase0EntityIDDelimiter = '|';
```

Comment:
@james-elastic @andrewstucki thoughts on a delimiter for the phase 0 entity_id construction? currently it is `endgame|<endpoint id>|<unique pid>`

Reply:
I'm fine with that, or something like `endgame-<unique pid>-<endpoint id>` where you could do a split on the first two `-` to recreate the tuple and validate that we're wanting the `endgame` special-snowflake data. I would remove all the references to `phase0*` though -- could call it something like `legacyEntityIDDelimiter`

Reply:
Cool, I'll switch everything to legacy.